File: CarAgency/ClientApp/src/models/RevisionTecnicaConcepto.ts
import RevisionTecnicaConceptoTipo from "models/RevisionTecnicaConceptoTipo";

export default class RevisionTecnicaConcepto {
    
    RevisionTecnicaConceptoId?: number;    
    Descripcion?: string;
    Tipo?: RevisionTecnicaConceptoTipo;

    constructor(){
        this.RevisionTecnicaConceptoId = undefined;
        this.Descripcion = undefined;
        this.Tipo = undefined;
    }

}

File: Domain/Models/Vehiculos/RevisionTecnica/RevisionTecnicaConcepto.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Models.Vehiculos
{
    public class RevisionTecnicaConcepto : ClaseBase
    {
        [Key]
        public int RevisionTecnicaConceptoId { get; set; }
        [Column(TypeName = "nvarchar(50)")]
        public string Descripcion { get; set; }
        public RevisionTecnicaConceptoTipo Tipo { get; set; }
    }
}
